pysat/instruments/methods/testing.py

```diff
@@ -29,3 +76,49 @@ def generate_fake_data(t0, num_array, period=5820, data_range=[0.0, 24.0],
         data = ((t0 + num_array) / period).astype(int)
 
     return data
+
```

Comment:
cyclic should have an explanation of what it does though it technically was not modified in this pull.